src/DurableTask.SqlServer/DurableTask.SqlServer.csproj

```diff
@@ -21,7 +21,7 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.10.0" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.11.1" />
```

Comment:
Would it make sense for us to replace the version with `2.*`? My hope is that this would reduce the need to keep the version up to date.

Reply:
Hmm...I used to think this didn't work, but reading the [documentation](https://learn.microsoft.com/en-us/nuget/concepts/package-versioning#floating-version-resolutions) again, I don't see any reason why it wouldn't. This would be a good thing for us to revisit across all of our projects, including those AssemblyInfo.cs files that we use in .NET Isolated projects.

Reply:
@cgillum: nice! So do we want to introduce that change here or punt it for a separate PR?

Reply:
I went ahead and introduced this change in another PR.